Doc/whatsnew/3.13.rst

```diff
@@ -185,6 +185,11 @@ Other Language Changes
 
   (Contributed by Sebastian Pipping in :gh:`115623`.)
 
+* When :func:`asyncio.TaskGroup.create_task` is called on an inactive
+  :class:`asyncio.TaskGroup`, the given coroutine will be closed (which
+  prevents a :exc:`RuntimeWarning`).
```

Comment:
I'd clarify that the RuntimeWarning is about "coroutine 'blank' was never awaited".

Reply:
Thanks for the comments! I've updated the wording